pkg/workspace/data_values_pre_processing.go

```diff
@@ -37,46 +38,89 @@ func (o DataValuesPreProcessing) Apply() (*DataValues, []*DataValues, error) {
 }
 
 func (o DataValuesPreProcessing) apply(files []*FileInLibrary) (*DataValues, []*DataValues, error) {
-	values := o.loader.schema.AsDataValue()
-	var libraryValues []*DataValues
-	for _, fileInLib := range files {
-		valuesDocs, err := o.templateFile(fileInLib)
-		if err != nil {
-			return nil, nil, fmt.Errorf("Templating file '%s': %s", fileInLib.File.RelativePath(), err)
-		}
+	_, checkSchema := o.loader.schema.(*schema.DocumentSchema)
```

Comment:
accessing private data from loader got me curious... it seems that template loader does not use schema at all. it seems weird for template loader to know anything about schema since it's not dealing with data values. i would have expected DataValuesPreProcessing to be initialized with appropriate schema.